lib/app/features/users/data/models/user_detail_profile_model.dart

```diff
@@ -8,23 +9,65 @@ class UserDetailProfileModel extends UserDetailProfileEntity {
     String? miniBio,
     String? skills,
     String? activity,
+    List<UserDetailBadgeEntity>? badges,
   }) : super(
           nickname: nickname,
           avatar: avatar,
           clientId: clientId,
           miniBio: miniBio,
           skills: skills,
           activity: activity,
+          badges: badges,
         );
 
   factory UserDetailProfileModel.fromJson(Map<String, dynamic> jsonData) {
+    List<UserDetailBadgeEntity>? badges;
+
+    if (jsonData['badges'] != null) {
+      badges = _parseBadges(jsonData['badges']);
+    }
     return UserDetailProfileModel(
       nickname: jsonData['apelido'],
       avatar: jsonData['avatar_url'],
       clientId: jsonData['cliente_id'],
       skills: jsonData['skills'],
       activity: jsonData['activity'],
       miniBio: jsonData['minibio'],
+      badges: badges,
```

Comment:
O que acontece aqui se o elemento `jsonData['badges']` for `null` e não entrar no if acima?

Reply:
A variável badges ja é nula, ela so vai ser preenchida se badges vier do backend, do contrario ela vai nulla e na page, se vier nulla, nao mostra nenhuma badge.

Reply:
Passou batido aqui que `badges` estava definido na linha 24.